packages/commons/src/events/test.utils.ts

```diff
@@ -139,6 +139,7 @@ export function mapFieldTypeToMockValue(
     case FieldType.PAGE_HEADER:
     case FieldType.LOCATION:
     case FieldType.SELECT:
+    case FieldType.SELECT_DATE_RANGE:
```

Comment:
Was this moved on purpose?

Reply:
Yes, from a rebase with develop it was not appearing